### DISCUSSION TITLE
difference in touch behaviour between hacker and non-hacker

### DISCUSSION THREAD

**Discussion by loveshack** [solokeys/solo2#102] [Q&A]
Should I expect a difference between hacker and non-hacker versions in the behaviour on touch?  While I hold the contact on a hacker one, the LED goes blue, but it stays green on non-hacker.  It's the same with .8 or .9 firmware in both.

(I'd rather not have the really bright -- unless it's in the black case -- light on all the time, only briefly when it's plugged in, and when it's indicating input; perhaps at least be able to make it a lot dimmer, if not intermittent.)

**Comment by PeterFalken** [comment]
I see the same behavior on my solo keys as well - the hacker version and the USB-A version the LED turns blue when I touch the gold contacts. But for two USB-C non-hacker version keys the led stays green always - same behavior for firmware versions .7, .8 & .9

**Reply by loveshack** [reply]
I didn't notice originally, but another difference is that the light on the hacker version fades in and out. I hope that indicates it's possible to change from the very bright full on default.

**Reply by PeterFalken** [reply]
Yes - the LED has a 'breathing' patter to it on my USB-C hacker and the USB-A version. But that light behavior is missing on the non-hacker USB-C keys.
